test/support/coronavirus_landing_page_steps.rb

```diff
@@ -31,6 +31,10 @@ def given_there_is_a_content_item_with_live_stream_enabled_and_ask_a_question_en
     stub_content_store_has_item(CORONAVIRUS_PATH, content_item_with_live_stream_enabled_and_ask_a_question_enabled)
   end
 
+  def given_there_is_a_content_item_with_live_stream_enabled_and_popular_questions_link_enabled
```

Comment:
related to the comment above, maybe rename this `given_there_is_a_content_item_with_popular_questions_link_enabled`

Reply:
✅